src/main/java/View/Web/Servlets/Privileged/UserSpecific/FlatInformationServlet.java

```diff
@@ -0,0 +1,63 @@
+package View.Web.Servlets.Privileged.UserSpecific;
+
+import Control.ConfigClass;
+import Control.Scenario.UC_FlatSummary;
+import Model.Database.Support.CustomLogs;
+import Model.Web.User;
+import View.Configuration.ContextUtil;
+import View.Support.DcsWebContext;
+import View.Support.ServletAbstracts.SessionServlet;
+import View.Support.ServletHelper;
+import View.Support.SessionUtil;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.WebContext;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "FlatInformationServlet", urlPatterns = FlatInformationServlet.SERVLET_URL)
+public class FlatInformationServlet extends SessionServlet {
+    public static final String SERVLET_URL =  "/action/projects/flats";
+    public static final String TEMPLATE_NAME = "views/privileged/my_flat.html";
```

Comment:
tu by som mozno zmenila nazov html ale to nie je dolezite

Reply:
Dovod?